teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+# inherite Person class
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(specialization)
```

Comment:
- [ ] Same remark about the parameters passed in initialize method, please kindly fix it as well 🙃 🙃

Reply:
Dear @cynthiainga thank you for the kind feedback. I have been able to realize the error it can cause without the parameters. I have applied the changes required successfully.